Tools/TestRunnerShared/TestFeatures.cpp

```diff
@@ -105,9 +105,8 @@ static std::optional<double> overrideDeviceScaleFactorForTest(const std::string&
     return std::nullopt;
 }
 
-static bool shouldDumpJSConsoleLogInStdErr(const std::string& pathOrURL)
+static bool isValidWPTURL(const URL& url)
```

Comment:
What does it mean to be a "valid" WPT url?

Reply:
Oh I'm mainly trying to say this is both a valid `URL` and is running on one of the ports where WPTs are expected to run, so for the lack of a better term went with `isValidWPTURL`. Thoughts?

Reply:
`isWPTTestURL` perhaps.

Reply:
Test is redundant with the acronym WPT

Reply:
Renamed to `isWebPlatformTestURL`.